php-ts-mode.el

```diff
@@ -76,10 +76,10 @@
     "elseif" "enddeclare" "endforeach" "endif" "endswitch"
     "endwhile" "enum" "extends" "final" "finally" "for" "foreach"
     "fn" "function" "global" "if" "implements" "include_once"
-    "include" "insteadof" "interface" "namespace" "never" "new"
```

Comment:
`never` is not keyword.

https://github.com/tree-sitter/tree-sitter-php/blob/1a40581b7a899201d7c2b4684ee34490bc306bd6/grammar.js#L505